.github/PULL_REQUEST_TEMPLATE/bugfix.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug fix
+about: Create a PR to improve this project
+title: ''
+labels: triage
+assignees: ''
+
+---
+
+**What issue is this PR fixing**
+Example:
+`closes #123`
+`fixes #456`
+
+**What is being changed**
+A clear description of what this PR brings.
+
+**Quality**
+* [ ] I successfully ran `yarn`, `yarn build`, `yarn test`, `yarn test:browser` locally
+* [ ] I allow my PR to be updated
+* [ ] I added unit tests
+* [ ] I added integration tests
+* [ ] I did not add automated tests because _________, and I am aware that a PR without tests will likely get rejected.
```

Comment:
All the items in this check list are very important in order to have the PR accepted. So, I think this item is not necessary.

Reply:
I guess I wasn't thinking about it as a checklist, but more like "check all the boxes that apply".

Good catch, I should make that more clear.

I want to keep this last item because I want to nudge folks into adding tests for everything and understanding in advance why their PRs might not be accepted.

Plus, it takes precious energy and time to ask for tests when there are none.